web-services/model/src/main/java/datawave/webservice/query/model/ModelBean.java

```diff
@@ -110,7 +124,12 @@
     @GZIP
     @Interceptors(ResponseInterceptor.class)
     public ModelList listModelNames(@QueryParam("modelTableName") @DefaultValue(DEFAULT_MODEL_TABLE_NAME) String modelTableName) {
-        ModelList response = new ModelList();
+        
+        if (modelTableName == null) {
+            modelTableName = defaultModelTableName;
+        }
+        
+        ModelList response = new ModelList(modelTableName, dataTablesUri, defaultModelTableName);
```

Comment:
You shouldn't be passing in `defaultModelTableName` here. It should be `modelTableName` so that if the user calls this method with something other than the default model table name, it will pass through and be used in the generated links.